.example.env

```diff
@@ -27,6 +24,12 @@ VEDA_DB_PUBLICLY_ACCESSIBLE=TRUE
 VEDA_RASTER_ROOT_PATH=
 VEDA_STAC_ROOT_PATH=
 
+VEDA_USERPOOL_ID=
+VEDA_CLIENT_ID=
+VEDA_CLIENT_SECRET=secret
+VEDA_DATA_ACCESS_ROLE_ARN=
+VEDA_COGNITO_DOMAIN=
+
```

Comment:
While you're modifying this file, can you also add:
- VEDA_DB_USE_RDS_PROXY=[OPTIONAL]
- VEDA_DB_RDS_INSTANCE_CLASS=[OPTIONAL]
- VEDA_DB_RDS_INSTANCE_SIZE=[OPTIONAL]

?

Reply:
Done